packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -465,16 +465,38 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
    *
    * See: https://docs.thinkboxsoftware.com/products/deadline/10.1/1_User%20Manual/manual/event-spot.html for additonal information.
    *
-   * @param includeResourceTracker Whether or not the Resource tracker admin policy should also be addd (Default: True)
+   * @param includeResourceTracker Whether or not the Resource tracker admin policy should also be added (Default: True)
+   * @param iamFleetRoleArns Allows render queue to use custom spot fleet roles (Default: only aws-ec2-spot-fleet-tagging-role is allowed)
    */
-  public addSEPPolicies( includeResourceTracker: boolean = true): void {
+  public addSEPPolicies(includeResourceTracker: boolean = true, iamFleetRoleArns: string[] | undefined = undefined): void {
```

Comment:
This is weird, I agree :)
It might be that initially I wrote `iamFleetRoleArns: string[] | undefined = []`, and then you can't have `?` and `=` at the same time. And then I decided that `[]` is not needed and it evolved into this.

Anyway, this policies are now added inside of `ConfigureSpotEventPlugin` construct, so I reverted changes to this function.